src/ompl/geometric/planners/bitstar/BITstar.h

```diff
@@ -397,9 +407,6 @@ namespace ompl
             ///////////////////////////////////////
             // Planner progress property functions
             /** \brief Retrieve the best exact-solution cost found
-            as the raw data. (bestCost_) */
```

Comment:
I'm going to defer.
I can tell you a fun story sometime about a day spent learning the hard way that it is safer for delete functions for a shared pointer to be called by value. :wink:

Essentially, a delete function that does nontrivial cleanup (i.e., both accessing the pointer and removing it from owning containers) to delete the pointer before it's done with it.

I'm not sure if it's this exact function that requires the pointer by value, but one of the `BIT*` functions definitely does, so I used the same pattern for all of them to be safe.